DOL.WHD.Section14c.Web/src/modules/components/changePasswordForm/changePasswordFormController.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = function(ngModule) {
+    ngModule.controller('changePasswordFormController', function($scope, $location, stateService, apiService) {
```

Comment:
This should list the dependencies as strings in an array with the function as the last element in the array.  This ensures dependency injection will keep working if variable names are mangled in the build process (e.g., minification).  See the [Angular guide to dependency injection](https://docs.angularjs.org/guide/di).